src/goto-instrument/goto_instrument_parse_options.h

```diff
@@ -79,7 +80,12 @@ Author: Daniel Kroening, kroening@kroening.com
   "(undefined-function-is-assume-false)" \
   "(remove-function-body):"\
   "(splice-call):" \
-
+  "(aggressive-slice)" \
+  "(call-depth):" \
+  "(harness-generator):" \
```

Comment:
I may have missed it, but what does this option do?

Reply:
Ah, that shouldn't be in this pull request.

Reply:
OK.